src/ExtensionsSample/ErrorNotifier.cs

```diff
@@ -21,18 +18,9 @@ public class ErrorNotifier
         private const string NotificationUriSettingName = "AzureWebJobsErrorNotificationUri";
         private readonly string _webNotificationUri;
         private readonly HttpClient _httpClient = new HttpClient();
-        private readonly SendGridAPIClient _sendGrid;
```

Comment:
For all of the extensions being removed there will be code like this littered through samples/tests. It will be very important when the code is readded to make sure nothing is missed. What strategy do you recommend people to follow to ensure this?